core/src/main/java/dk/alexandra/fresco/framework/sce/evaluator/BatchedProtocolEvaluator.java

```diff
@@ -73,12 +73,12 @@ public void eval(ProtocolProducer protocolProducer, ResourcePoolT resourcePool,
       } else {
         zeroBatches = 0;
       }
-/*
+      /*
```

Comment:
We should look into this issue at some point. Maybe make an issue on it. It should be a bug if empty batches appear often, but apparently this has been removed for a reason. Should be dealt with somehow (not in this PR)